### ISSUE TITLE
Generate Particle Manifest creates bad paths

### ISSUE THREAD

**Issue by bottiger1** [ruarai/CompilePal#122]
**Describe the bug**
If you use Generate Particle Manifest, it will create an invalid path for your particle files.

It will also pack another copy of this file with your tf2 map path.

![bug](https://user-images.githubusercontent.com/55270538/88341729-facb4980-ccf2-11ea-941b-bb1b1a1e62f2.jpg)

**Steps to reproduce**
Use a particle file in your custom folder and add Utility and Generate Particle Manifest.

It will incorrectly generate this particle manifest file.

```
particles_manifest
{
      "file"    "!custom\ze_skyrim_v12_fix2\particles\fusrodah.pcf"
      "file"    "!custom\ze_skyrim_v12_fix2\particles\fusroderp.pcf"
}

```

**Expected behavior**

```
particles_manifest
{
      "file"    "particles\fusrodah.pcf"
      "file"    "particles\fusroderp.pcf"
}
```
[debug.log](https://github.com/ruarai/CompilePal/files/4968953/debug.log)

Version: v027.11

**Comment by Exactol** [comment]
I think this has been fixed by 1551e2e, just forgot to close the issue. If it still isn't fixed, feel free to reopen this issue.

**Comment by bottiger1** [comment]
> I think this has been fixed by [1551e2e](https://github.com/ruarai/CompilePal/commit/1551e2e4f4b0e79444b9f1781f6e312c23941cb6), just forgot to close the issue. If it still isn't fixed, feel free to reopen this issue.

It was fixed by my commit you just merged. I tested it and it was still packing a duplicate manifest so I remade the fix in my new commit.
